docs/usage/18-contrib/0-open-telemetry.md

```diff
@@ -0,0 +1,34 @@
+# Open Telemetry Instrumentation
+
+Starlite includes optional Open Telemetry instrumentation that is exported from `starlite.contrib.open_telemtry`. To use
+this package, you should first install the required dependencies:
+
+```shell title="as separate packages"
+pip install opentelemetry-sdk opentelemetry-instrumentation-asgi
```

Comment:
SDK is not a required dependency for the library authors like you (although you need it for testing). The instrumentation only uses the `opentelemetry-api` and leaves the choice of SDK to the user. I think you are also not using SDK in instrumentation.

Reply:
aight, thanks!